schutzbot/Jenkinsfile

```diff
@@ -1,8 +1,10 @@
+var cron_string = BRANCH_NAME == "main" ? "@hourly" : ""
```

Comment:
Why `@hourly` ? IMO this should be `@daily`

Reply:
I thought you would be happy with more testing...

Yeah, sorry, my copy-paste error, fixed. :)

Reply:
> I thought you would be happy with more testing...

I am but we don't get new builds every hour :-).